app/services/case_remove_sar_deadline_extension_service.rb

```diff
@@ -0,0 +1,28 @@
+class CaseRemoveSARDeadlineExtensionService
+  attr_reader :result, :error
+
+  def initialize(user, kase)
+    @user = user
+    @case = kase
+    @result = :incomplete
+  end
+
+  def call
```

Comment:
Not really sure about the boiler-plate error handling. Most services catch specific errors, this one (and the one above) catch everything (even syntax errors)

Reply:
Think the transaction handling could do with documenting and/or splitting out (probably not in this story though). I think there is a general desire to not show the standard crash page when things go wrong, but to rollback the transaction and be polite.

Reply:
Discussed with @starswan - leave as is for now as the handling of errors is a system wide issue requiring a broader discussion